package.json

```diff
@@ -89,6 +89,7 @@
     "react-mapbox-gl": "^5.1.0",
     "react-mapbox-gl-draw": "^2.0.4",
     "react-player": "^2.8.2",
+    "react-use": "^17.2.1",
```

Comment:
This is another (big) package - is it necessary? https://bundlephobia.com/result?p=react-use@17.2.1

Reply:
I'll look for some solution to it.

Reply:
Generally it is no problem if we add a package if it is the best solution for the issue and everything else is too difficult. We just should have a short check before, if the problem can not be solved easily without it.